### PR TITLE
Single food endpoint

### LINKED ISSUE #2: User can get a single Food from database
```
As a user
I should be able to GET request 'api/v1'foods/:id`
Where ID is the valid ID of a Food resource
And receive the information for only that Food
```
Example response:
GET `api/v1/foods/1`
```
{
    "id": 1,
    "name": "Banana",
    "calories": 150
}
```

### PR DESCRIPTION
**Pull request by tnodland**
Resolves #2 

Adds functionality for the GET '/api/v1/foods/:id' route, returning a single food object from the database.

Test setup with Jest complete, test is passing, but tests fail to close once run

### CODE REVIEW

**Comment by BrennanAyers** [comment]
Maybe we should add Node_modules to the .gitignore? @tnodland

**Comment by tnodland** [comment]
> Maybe we should add Node_modules to the .gitignore? @tnodland

Yeah oops 🤣

**Comment by BrennanAyers** [comment]
I'm going to take a look into why Travis is not appreciating what we're doing here!

**Comment by BrennanAyers** [comment]
@tnodland Hey, can you add this to the travis config file, so we can see if it will pass after creating a database?
```
services:
  - postgresql
before_script:
  - psql -c 'CREATE DATABASE travis_test;' -U postgres
```

File: config/config.json [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+    "username": "postgres",
     "password": null,
     "database": "quantified_shelf_development",
     "host": "127.0.0.1",
     "dialect": "postgres",
     "operatorsAliases": false
   },
   "test": {
-    "username": "root",
+    "username": "postgres",
```
Comment (BrennanAyers) [inline config/config.json]: This worked!

**Pull request merged**